com.governikus.ausweisapp2.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: com.governikus.ausweisapp2
+runtime: org.kde.Platform
+runtime-version: "5.14"
+sdk: org.kde.Sdk
+command: AusweisApp2
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pcsc
+  - --own-name=org.kde.* # Workaround for org.kde.StatusNotifierWatcher support, see https://github.com/flathub/com.viber.Viber/issues/4
```

Comment:
You probably need `--talk-name=org.kde.StatusNotifierWatcher` too.

Reply:
Added.